extensions/git-branch-name-generator/src/index.tsx

```diff
@@ -1,56 +1,73 @@
-import { Action, ActionPanel, Clipboard, Form, popToRoot, showHUD, showToast, Toast } from "@raycast/api";
-import { useForm } from "@raycast/utils";
+import { Form, ActionPanel, Action, showToast, Clipboard } from "@raycast/api";
 
 type Values = {
-  branch_name: string;
-  type_of_branch: string;
+  branchType: string;
+  prefix: string;
+  description: string;
+  snakeCase: boolean;
+  prefixDate: boolean;
 };
 
+function slugify(content: string, separator: string) {
+  return content.toLowerCase().replaceAll(" ", separator);
+}
+
 export default function Command() {
-  const { handleSubmit, itemProps } = useForm<Values>({
-    async onSubmit(values) {
-      const branchType = values.type_of_branch ? `${values.type_of_branch}/` : "";
-      const branchNameAsSlug = values.branch_name
-        ? values.branch_name
-            .toLowerCase()
-            // replace all but 0-9, a-z and / with a dash
-            .replace(/[^0-9a-z/]/g, "-")
-            // replace multiple dashes with a single dash
-            .replace(/[-]+/g, "-")
-            // trim dashes from the beginning
-            .replace(/^-+/, "")
-            // trim dashes from the end
-            .replace(/-+$/, "")
-            // only take the first 100 characters
-            .substring(0, 100)
-        : "";
-      const fullBranchName = `${branchType}${branchNameAsSlug}`;
-      Clipboard.copy(fullBranchName);
-      await showHUD(`Copied to clipboard: ${fullBranchName}`);
-      popToRoot({ clearSearchBar: true });
```

Comment:
Can we add `popToRoot({ clearSearchBar: true })` back? It's pretty annoying that it persists after I've generated a branch name

Reply:
yea got it back :)